setup.sh

```diff
@@ -70,6 +70,7 @@ pssh --inline \
     --host "$MASTERS $SLAVES" \
     --user root \
     --extra-args "-t -t $SSH_OPTS" \
+    --timeout 300 \
```

Comment:
Ok - The TCPKeepAlive (essentially wait for the process to finish until TCP connections are up)) sounds similar to setting timeout 0 for pssh. So I'd suggest doing that. 
@tomerk It'll be great if you can test if `timeout = 0` works correctly.